<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1168</width>
    <height>681</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="imageViewer">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>721</width>
      <height>591</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QListWidget" name="imageList">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>261</width>
      <height>322</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="cropButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>130</y>
      <width>40</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="rotateRightButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>180</y>
      <width>40</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="rotateLeftButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>230</y>
      <width>40</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="flipButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>280</y>
      <width>40</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>80</y>
      <width>40</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="openButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>40</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="filter1Button">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>20</y>
      <width>54</width>
      <height>48</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="filter1Label">
    <property name="geometry">
     <rect>
      <x>1107</x>
      <y>75</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Filter 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="filter2Label">
    <property name="geometry">
     <rect>
      <x>1107</x>
      <y>155</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Filter 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="filter2Button">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>100</y>
      <width>54</width>
      <height>48</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="filter3Label">
    <property name="geometry">
     <rect>
      <x>1107</x>
      <y>235</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Filter 3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="filter3Button">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>180</y>
      <width>54</width>
      <height>48</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="filter4Label">
    <property name="geometry">
     <rect>
      <x>1107</x>
      <y>315</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Filter 4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="filter4Button">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>260</y>
      <width>54</width>
      <height>48</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="redRGBButton">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>450</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="blueRGBButton">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>500</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="greenRGBButton">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>550</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="histogramViewer">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>341</width>
      <height>251</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1168</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../Resources/MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
